src/services/DeploymentService.ts

```diff
@@ -0,0 +1,364 @@
+import { Headers, RequestAPI, get, post } from "request";
+import { getRequest, postRequest, uploadFile } from "./Service";
+
+export interface App {
+    AppId: string;
+    Name: string;
+    ProjectId: string;
+    Url: string;
+}
+
+export interface Job {
+    JobId: string;
+}
+
+export interface Environment {
+    Status: string;
+    Url: string;
+    mode: EnvironmentMode;
+}
+
+export interface EnvironmentStatus {
+    Status: "Starting" | "Started";
+}
+
+export interface Upload {
+    PackageId: string;
+}
+
+export interface Sandbox {
+    AppId: string;
+    Name: string;
+    ProjectId: string;
+    Url: string;
+}
+
+export interface Branch {
+    Name: string;
+    DisplayName: string;
+    LatestTaggedVersion: string;
+    LatestRevisionNumber: number;
+    LatestRevisionMendixVersion: string;
+}
+
+export interface EnvironmentSettings {
+    Constants: Constant[];
+    CustomSettings: CustomSetting[];
+    ScheduledEvents: ScheduledEvent[];
+}
+
+export interface Constant {
+    Name: string;
+    DataType: string;
+    Value: string;
+    DeployedValue: string;
+}
+
+export interface CustomSetting {
+    Name: string;
+    Value: string;
+}
+
+export interface ScheduledEvent {
+    Name: string;
+    Value: string;
+    DeployedValue: string;
+}
+
+export type EnvironmentMode = "Sandbox" | "Test" | "Acceptance" | "Production";

```

Comment:
If the type definitions are becoming too many, may need to extract them into their own .d.ts